### DISCUSSION TITLE
第三版公式16.1求助

### DISCUSSION THREAD

**Discussion by LittleTheFu** [kanition/pbrtbook#184] [交流]
实在是不好意思，看书实在是看不明白了，搜到了这里，不知道在这里提问是否合适？

https://www.pbr-book.org/3ed-2018/Light_Transport_III_Bidirectional_Methods/The_Path-Space_Measurement_Equation
这个是原书的链接，我关于公式有看不懂的地方。

如下图，红线部分，我不明白，为什么AFile忽然就没有了，就变成A了？

![捕获](https://github.com/user-attachments/assets/0a709620-bff3-4666-bbe5-92b3b31aad3d)

**Comment by kanition** [comment]
抱歉，我现在翻译进度在第8章，所以第16章的内容我也不太了解。这个公式具有上下文背景，所以也不能比较快速地纯粹从数学角度推导出来，我帮不了什么忙。这个问题可以一直挂在这里，也许有其他同好看见了可以帮忙解答。讨论区一直非常冷清，也可能没人能解答

**Reply by LittleTheFu** [reply]
谢谢你的回复，也祝你这个项目能顺利进行下去。

**Comment by kanition** [comment]
你也可以查询讨论区或issue区的历史记录，看有没有合适的同好可以询问。有人还建了讨论群，也许可以试试。
